### PR TITLE
Windows Batch and Powershell Test Installation files added

### PR DESCRIPTION
**Pull request by th-neu**
added a batch and powershell script to test the installation

### CODE REVIEW

**Review by keldenl — CHANGES_REQUESTED** [review]
hey i tried running the .bat file in windows and after inputting my file the whole cmd window just disappears

i tried with proper .bin file and with random text, it just disappears.. do you know what might be the issue?

**Comment by th-neu** [comment]
Yes pause was missing. And i cleaned the file. To much "REM" add the end.

**Comment by keldenl** [comment]
@th-neu  works better, but i'm stuck on "can't find file in \llama.cpp\models\". i think it's because it's looking for it in the absolute path (\llama.cpp\models), but the following should be allowed: (C:\\documents\llama.cpp\models\). 

also the powershelgl script isn't waiting either, so whatever you did for fix it for cmd u should apply to the powershell too

**Comment by th-neu** [comment]
Scripts have been updated

File: test-installation.bat [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+) else (
+    echo.
+    echo Error: Curl command failed!
+    echo Is the gpt-llama.cpp server running? Try starting the server and running this script again.
+    echo Please check for any errors in the terminal window running the gpt-llama.cpp server.
+)
+
+exit /b %errorlevel%
+
+pasue
```
Comment (keldenl) [inline test-installation.bat:58]: typo

File: test-installation.bat [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+REM Check if the file has a .bin extension
+set "ext=%modelpath:~-4%"
+if not "%ext%" == ".bin" (
+    echo Error: The file extension is not .bin. Please make sure you have provided the correct file.
+    pause
+    exit /b 1
+)
+
+set "validpath=llama.cpp\models\"
```
Comment (keldenl) [inline test-installation.bat:23]: this still didn't work for me. also please update the error message to remove `\`

**Pull request closed without merging**